public/utils/helpers.tsx

```diff
@@ -242,3 +242,12 @@ export const getPlugins = async (opensearchService: OpenSearchService) => {
     return [];
   }
 };
+
+/**
+ * Returns last route param based on route path
+ * @param {string} pathname
+ * @param {string} route
+ * @return {string}
+ */
+export const getRouteParam = (pathname, route): string =>
```

Comment:
Instead of creating our own helper can we not use existing utilities from react-router itself? for e.g. https://stackoverflow.com/a/70446743

Reply:
Done, introduced the `withRouter` fn from `react-router` and URL parameters are extracted from `match.params`